MoA/models/llama/modeling_llama.py

```diff
@@ -219,14 +207,15 @@ class LlamaMixtureAttention(LlamaAttention):
     def forward(
         self,
         hidden_states: torch.Tensor,
-        attention_mask: Optional[torch.LongTensor] = None,
+        attention_mask: Optional[torch.Tensor] = None,
         position_ids: Optional[torch.LongTensor] = None,
-        past_key_value: Optional[StaticCircularCache] = None,
+        past_key_value: Optional[Cache] = None,
```

Comment:
I think it should be `StaticCircularCache` and `LongTensor`. It would be best if you could help us double-check

Reply:
You are right. I replace Cache with StaticCircularCache in next commit.